test/README.md

```diff
@@ -82,6 +82,45 @@ environment.
    docker is used only for running drenv tests locally. You can use
    podman for building and running containers locally.
 
+1. Install `podman`
+
+   ```
+   sudo dnf install podman
+   ```
+
+   ramenctl depends on podman, hence it needs to be installed.
```

Comment:
We typically don't describe why something is neded, lets use the same content added here:
https://github.com/RamenDR/ramen/blob/main/docs/user-quick-start.md